index.js

```diff
@@ -58,6 +62,7 @@ function listen() {
         }
     },
     () => {
+        logging.addmetric("category", "action", "label", 987654321);
```

Comment:
Category: type of event (i.e mouse click)
Action: specific event  (right click, left click, etc...)
Label: String related with that event
Value: Value of the event